homebrew_releaser/formula.py

```diff
@@ -50,6 +51,14 @@ def generate_formula_data(
         if first_word_of_desc[0].lower() in articles:
             description = first_word_of_desc[1].strip().capitalize()
 
+        depends_on_content = ''
+
+        if depends_on:
+            dependencies = [dependency.strip() for dependency in depends_on.split('\n') if dependency]
```

Comment:
I'm not sure the `if dependency` is needed here since if there is nothing present, it'll be an empty list anyway and iterating an empty list won't do anything. Please let me know though if I've missed something or misunderstood.

```suggestion
            dependencies = [dependency.strip() for dependency in depends_on.split('\n')]
```

Reply:
This was because the string was not stripped, I updated the condition to simplify it by calling `depends_on.strip().split('\n')`.

Before that `depends_on.split('\n')` was returning: `['', '"gcc"', '"bash" => :build', '']`

Reply:
This is due to the variable declaration at the top of the `formula_test` file:

```python
DEPENDS_ON = """
"gcc"
"bash" => :build
"""
```

Maybe you have a tip in Python to avoid new lines ? I'm a newcomer to Python ;-)

Reply:
Ah that makes sense, in that case, you'll definitely want to put this line back to `dependencies = [dependency.strip() for dependency in depends_on.split('\n') if dependency]`. What you changed it to won't work as it's going to strip the whole string and not each individual piece after splitting on the newline.